src/app/shared/example-viewer/example-viewer.scss

```diff
@@ -21,23 +25,31 @@
   flex: 1 1 auto;
 }
 
-.docs-example-source-copy {
-  position: absolute;
-  top: 8px;
-  display: none;
+.docs-example-viewer-body {
+  padding: 30px;
 }
 
-.docs-example-source-wrapper:hover {
-  .docs-example-source-copy {
-    display: inline-block;
-  }
+.button-bar {
+  float: right;
+}
+
+code-snippet {
+  padding: 20px;
+  background: rgba(0, 0, 0, 0.01);
 }
 
 .docs-example-source {
-  padding: 0 30px 10px 30px;
-  min-height: 150px;
+  padding: 0;
```

Comment:
can we eliminate the source of these instead of zeroing them out?

Reply:
The source is from `_markdown.scss` but they also control the style of standalone snippets and removing them now makes them look weird. Can probably remove it once we remove all the standalone snippets

Reply:
can you leave a TODO for now?

Reply:
This comment is marked resolved but I don't see a TODO?

Reply:
oops didn't see the followup comment